package.json

```diff
@@ -85,7 +85,7 @@
     }
   },
   "scripts": {
-    "build": "shx rm -rf dist && tsc -b",
+    "build": "shx rm -rf dist && tsc -b && shx cp src/lib/phatjs/phatjs_bg.wasm dist/lib/phatjs/",
```

Comment:
Name the file with version or commit hash suffix, e.g. `phatjs_bg.0.1.0.wasm` or `phatjs_bg.xxxxxx.wasm`